src/treetensornetworks/opsum_to_ttn.jl

```diff
@@ -59,8 +59,10 @@ function ttn_svd(
   vertextype_sites = vertextype(sites)
   thishasqns = any(v -> hasqns(sites[v]), vertices(sites))
 
+  linkdir_ref = ITensors.using_auto_fermion() ? ITensors.In : ITensors.Out
```

Comment:
This seems strange to me, could you explain this? Would at least be good to put an explanation as a comment so we can understand it later on.

Reply:
I see, so basically it is just a matter of what convention we will use for the index directions of the link indices of the TTNO, and one of them is more convenient for the autofermion system?

I don't care about that convention either way, so if one is simpler/better for the autofermion system we could just go with that.

Reply:
I went with `linkdir_ref = ITensors.using_auto_fermion() ? ITensors.In : ITensors.In` to indicate that this may still be changed, and also added a comment. Happy to change it to `linkdir_ref = ITensors.In` instead.

Reply:
I would prefer the simpler version, and then leave a comment explaining it is set to `ITensors.In` since that works better with the autofermion system but is subject to change.